src/Rocket.cpp

```diff
@@ -13,13 +13,23 @@
 
 #include "Rocket.h"
 
-#include "Vector3.h"
-#include "quaternion.h"
+#include <Eigen/Dense>
+
+using Eigen::Quaterniond;
+using Eigen::Vector3d;
 
 Rocket::Rocket() {
-    q_ornt_ = Quaternion<double>(1, 0, 0, 0);
+    q_ornt_ = Quaterniond(1, 0, 0, 0);
```

Comment:
Honestly all of these variables (except `Cp_vect_`) can be initialized in the class definition in `Rocket.h`. Not sure why I put these here before, but I'd suggest doing it in the header file.

`Cp_vect_` still needs to be constructed here though, since some math needs to be done for it. Could you switch that to use **list initialization** with curly brackets `Vector3d{...}` for consistency?